<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="ParaTest\Tests\fixtures\failing_tests\UnitTestWithErrorTest" file="./test/fixtures/failing_tests/UnitTestWithErrorTest.php" tests="1" assertions="0" errors="1" warnings="0" failures="0" skipped="0" time="1.234567">
    <testcase name="testTruth" class="ParaTest\Tests\fixtures\failing_tests\UnitTestWithErrorTest" classname="ParaTest.Tests.fixtures.failing_tests.UnitTestWithErrorTest" file="./test/fixtures/failing_tests/UnitTestWithErrorTest.php" line="19" assertions="0" time="1.234567">
      <error type="RuntimeException">ParaTest\Tests\fixtures\failing_tests\UnitTestWithErrorTest::testTruth
RuntimeException: Error!!!

./test/fixtures/failing_tests/UnitTestWithErrorTest.php:21</error>
    </testcase>
  </testsuite>
</testsuites>
